docs/gui.rst

```diff
@@ -384,7 +384,7 @@ Image I/O
 
 .. function:: ti.imwrite(img, filename)
 
-    :parameter img: (Matrix or Expr) the image you want to export
+    :parameter img: (ti.Matrix.field or Expr) the image you want to export
```

Comment:
```suggestion
    :parameter img: (ti.Vector.field or ti.field) the image you want to export
```

Reply:
Curious: does `ti.imwrite` doesn't work for `ti.Matrix` anymore?

Reply:
It only works with `Matrix` with only one row, i.e., `Vector`.